tasks/src/main/java/com/crud/tasks/scheduler/EmailScheduler.java

```diff
@@ -28,7 +28,15 @@ public void sendInformationEmail() {
         simpleEmailService.send(new Mail(
                 adminConfig.getAdminMail(),
                 SUBJECT,
-                "Currently in database you have got: " + size + " tasks"
+                "Currently in database you have got: " + size + " " + createPluralWord("task", size)
         ));
     }
+
+    private String createPluralWord(String text, long size) {
+        if (size == 1) {
```

Comment:
lepiej uzyc trinary operator: return size == 1 ? text : text + "s";
Chyba odwrotnie :D bo jesli jest jeden, to bez s na koncu, jak Piskutas bez s